gluetool/utils.py

```diff
@@ -818,7 +814,7 @@ def __init__(self, filepath, logger=None):
             if not isinstance(pattern_dict, dict):
                 raise GlueError("Invalid format: '- <pattern>: <result>' expected, '{}' found".format(pattern_dict))
 
-            pattern = pattern_dict.keys()[0]
+            pattern = list(pattern_dict.keys())[0]
```

Comment:
Under Python 2, this would a list made of a list. `list(iterkeys(pattern_dict))` should work the same for both.

Reply:
thanks, fixed